grails-app/controllers/org/transmartproject/rest/StudyController.groovy

```diff
@@ -51,4 +55,18 @@ class StudyController {
     def show(String id) {
         respond studiesResourceService.getStudyById(id)
     }
+    
+    private def wrapStudies(Object source) {
+        
+        new ContainerResponseWrapper(
+                container: source,
+                componentType: Study,
+                links: [
+                        new Link(grails.rest.render.util.AbstractLinkingRenderer.RELATIONSHIP_SELF,
+                                "/studies"
+                        )
+                ]
```

Comment:
Style:

* `def` is redundant in `private def` (though it occurs in other methods in rest-api)
* no line after signature
* no need for the constant to be fully qualified and have a long line there. Either static import it or import the class.
* Be consistent in the line breaks. If you put the first argument immediately after the opening parenthesis, then you should put the closing parenthesis after the last argument, with no line break. Symmetry!